alvr/client_core/src/c_api.rs

```diff
@@ -703,11 +704,13 @@ pub struct AlvrStreamConfig {
     foveation_edge_ratio_y: f32,
 }
 
+/// Requires calling glMakeContext again after this function
 #[no_mangle]
 pub extern "C" fn alvr_initialize_opengl() {
     GRAPHICS_CONTEXT.set(Some(Rc::new(GraphicsContext::new_gl())));
 }
```

Comment:
cant we call glMakeContext in here itself, since its falloff of wgpu implementation? Also client_core is in android native context also.

Reply:
Well, if you end up calling another alvr rendering function right after, the glMakeCurrent call becomes redundant. In alvr_client_openxr instead we call glMakeCurrent just before we need it instead of just after the alvr calls.

Reply:
I don't want to be too nitpicky, since this C redering API is used only by PhoneVR. So if you request it i can make the glMakeCurrent calls in here. The only place where i can't is when destroying the RENDERING_CONTEXT, so you will need to either destroy your gl resources before that call, or call gmMakeCurrent yourself